src/benchmarks/micro/libraries/System.Diagnostics/Perf_Activity_Enumerations.cs

```diff
@@ -0,0 +1,186 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using System.Collections.Generic;
+
+namespace System.Diagnostics
+{
+    [MemoryDiagnoser]
+    [BenchmarkCategory(Categories.Libraries)]
+    public class Perf_Activity_Enumerations
```

Comment:
by moving the existing benchmarks to a new type with a new name you are changing the benchmark ID, which is used by our Reporting System for tracking performance over time. Please revert this change.

https://github.com/dotnet/performance/blob/master/docs/microbenchmark-design-guidelines.md#benchmarks-are-immutable